src/fidesops/graph/config.py

```diff
@@ -211,6 +211,19 @@ def is_member_of(self, collection_address: CollectionAddress) -> bool:
             and self.collection == collection_address.collection
         )
 
+    @staticmethod
+    def from_string(field_address_str: str) -> FieldAddress:
+        """Generate a field address from a string of the form 'dataset:collection:field_path'"""
+        try:
+            field_address_components = field_address_str.split(":")
+            # Ensure there are at least three components
+            field_address_components[2]  # pylint: disable=pointless-statement
+            return FieldAddress(*field_address_components)
+        except Exception:
+            raise FidesopsException(
+                f"'{field_address_str}' is not a valid field address"
+            )
+
```

Comment:
If it is kept it could be simplified slightly.

```suggestion
    @staticmethod
    def from_string(field_address_str: str) -> FieldAddress:
        """Generate a field address from a string of the form 'dataset:collection:field_path'"""
        field_address_components = field_address_str.split(":")
        # Ensure there are at least three components
        if len(field_address_components) < 2:
            raise FidesopsException(
                f"'{field_address_str}' is not a valid field address"
            )
        return FieldAddress(*field_address_components)
```

Reply:
I just removed!